staging/src/github.com/openshift/openshift-controller-manager/pkg/build/controller/build/build_controller.go

```diff
@@ -360,10 +373,57 @@ func (bc *BuildController) defaults() builddefaults.BuildDefaults {
 	return copy
 }
 
-func (bc *BuildController) setBuildDefaultProxy(spec *configv1.ProxySpec) {
+func (bc *BuildController) setBuildProxy(buildCfg *configv1.Build, globalProxy *configv1.Proxy) {
 	bc.configLock.Lock()
 	defer bc.configLock.Unlock()
-	bc.buildDefaults.DefaultProxy = spec
+
+	if buildCfg == nil && globalProxy == nil {
+		bc.buildDefaults.Config = nil
```

Comment:
Set the Git proxies to empty. `bc.buildDefaults.Config` includes the default environment variables, node selectors, etc - all of this currently comes from the openshift-controller-manager-operator

Reply:
part of next push